.github/.kube-linter-config.yaml

```
@@ -0,0 +1,9 @@
+
+checks:
+  # include explicitly adds checks, by name. You can reference any of the built-in checks.
+  # Note that customChecks defined above are included automatically.
+  include: [ ]
+  # exclude explicitly excludes checks, by name. exclude has the highest priority: if a check is
+  # in exclude, then it is not considered, even if it is in include as well.
+  exclude: [ "unset-cpu-requirements", "unset-memory-requirements", "no-read-only-root-fs", "run-as-non-root", "env-var-secret", "liveness-port",
```

Can we make it like this (or something similar if I missed something on the syntax)?
```suggestion
  exclude:
    - "unset-cpu-requirements"
    - "unset-memory-requirements"
    - "no-read-only-root-fs"
    - "run-as-non-root"
    - "env-var-secret"
    - "liveness-port"
```